src/components/Sum.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Sum = ({ a = 1, b = 2 }) => (
```

Comment:
If either give default values to your props, or indicate them as required in PropTypes.